CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing to Mainflux
```

Comment:
We do not need this file, you can remove contributing info from this repo (we have it in the main Mainflux repo)